backends/hip/ceed-hip-compile.cpp

```diff
@@ -39,9 +39,18 @@ int CeedCompileHip(Ceed ceed, const char *source, hipModule_t *module,
   hipFree(0); // Make sure a Context exists for hiprtc 
   hiprtcProgram prog;
 
-  // Add hip runtime include to string for generation
   std::ostringstream code;
-  code << "\n#include <hip/hip_runtime.h>\n";
+  // Add hip runtime include to string for generation if runtime < 40400000
+  // (implies ROCm < 4.5)
+  int runtime_version;
+  CeedChk_Hip(ceed, hipRuntimeGetVersion(&runtime_version));
+  if (runtime_version < 40400000)
```

Comment:
Minor style preference - in an if -else, when at least one of the two body statements requires {}s, I generally prefer to guard both bodies with {}s. It is just easier for me to read, personally.

Reply:
I added the {} (and tweaked the comments) in 682583ce514e65ab3543cf99b41ac809995991cd